test.js

```diff
@@ -31,12 +31,11 @@ test('check if what we generate equals to what translate.google.com generates',
 });
 
 test('support translate.google.cn via opts.tld', async t => {
-    config.clear();
-    const token = await getToken('hi', {tld: 'cn'});
```

Comment:
This is intentional change `hello` --> `hi` to avoid potential intersection with tests as token is stored for all tests.
I agree that config is clearer. But if for some reason clear operation will no be performed (new version of configstore appears or whatever), we should brake test.
I suggest to keep different phrases here.

Reply:
Alright, then. I'll resolve that while I'm on the hot streak.